CodeEdit/Features/WindowCommands/FileCommands.swift

```diff
@@ -44,12 +44,16 @@ struct FileCommands: Commands {
             .keyboardShortcut("w", modifiers: [.control, .shift, .command])
 
             Button("Close Window") {
-
+                NSApp.keyWindow?.close()
```

Comment:
Should the documentcontroller also be closed here if the window was the last one belonging to the document controller?

Reply:
no, because close window can also be used on the welcome and about screen, not just the workspace.

Reply:
but i can add it if you prefer 😄

Reply:
I'm actually not sure if it's needed or not, but it'd mostly be to make sure the document has been closed properly. If the window isn't of a documentcontroller, it should just close the window like you did right now.

Reply:
this operates the same way that CMD+Q works and saves + closes the file, resolving conversation